lib/google_apis/generator/elixir_generator/endpoint.ex

```diff
@@ -257,7 +257,7 @@ defmodule GoogleApis.Generator.ElixirGenerator.Endpoint do
 
   defp method_name_to_endpoint_name(method_name) do
     method_name
-    |> String.replace(".", "_")
+    |> String.replace(~r{[\.-]}, "_")
```

Comment:
does the `-` not need an escape character?

Reply:
No it doesn't.